plutus-core/plutus-core/src/PlutusCore/Builtin/KnownType.hs

```diff
@@ -245,10 +227,10 @@ class uni ~ UniOf val => MakeKnownIn uni val a where
     -- See Note [Cause of failure].
     -- | Convert a Haskell value to the corresponding PLC val.
     -- The inverse of 'readKnown'.
-    makeKnown :: Maybe cause -> a -> ExceptT (ErrorWithCause MakeKnownError cause) Emitter val
+    makeKnown :: Maybe cause -> a -> ExceptT (ErrorWithCause ReadKnownError cause) Emitter val
```

Comment:
This seems weird. Why does `makeKnown` throw a `ReadKnownError`? If we're just not distinguishing them any more, let's give it a better name.

Reply:
Because `readKnown` is now deferred to the same place where `makeKnown` happens. I'll change the name.